src/lib/util.ts

```diff
@@ -288,3 +288,39 @@ export function checkIfImageExists(url: string, callback: (exists: boolean) => v
 		};
 	}
 }
+
+export function linkIsVideo(link: string): boolean {
```

Comment:
You don't have to do this, but you could combine `linkIsVideo` and `linkIsLogfile` into a single `classifyVideo` function. You would need to use `{@const ...}` in the component so you don't have to call classify twice.